DigitalOffice/Tests/CompanyServiceTests/Repositories/CompanyRepositoryTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using LT.DigitalOffice.CompanyService.Database;
+using LT.DigitalOffice.CompanyService.Database.Entities;
+using LT.DigitalOffice.CompanyService.Repositories;
+using LT.DigitalOffice.CompanyService.Repositories.Interfaces;
+using LT.DigitalOffice.CompanyService.RestRequests;
+using LT.DigitalOffice.CompanyServiceUnitTests.UnitTestLibrary;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.CompanyServiceUnitTests.Repositories
+{
+    class CompanyRepositoryTests
+    {
+        private CompanyServiceDbContext dbContext;
+        private ICompanyRepository repository;
+
+        private DbPosition dbPosition;
+        private DbPosition newPosition;
+        private Guid positionId;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            var dbOptions = new DbContextOptionsBuilder<CompanyServiceDbContext>()
+                                    .UseInMemoryDatabase(databaseName: "InMemoryDatabase")
+                                    .Options;
+            dbContext = new CompanyServiceDbContext(dbOptions);
+            repository = new CompanyRepository(dbContext);
+        }
+
+        [SetUp]
+        public void SetUp()
+        {
+            positionId = Guid.NewGuid();
+            dbPosition = new DbPosition
+            {
+                Id = positionId,
+                Name = "Name",
+                Description = "Description"
+            };
+
+            dbContext.Positions.Add(dbPosition);
+            dbContext.SaveChanges();
+        }
+
+        [TearDown]
+        public void CleanDb()
+        {
+            if (dbContext.Database.IsInMemory())
+            {
+                dbContext.Database.EnsureDeleted();
+            }
+        }
+
+        [Test]
+        public void SuccessfullyEditPosition()
+        {
+            newPosition = new DbPosition
+            {
+                Id = positionId,
+                Name = "abracadabra",
+                Description = "bluhbluh"
+            };
+
+            var result = repository.EditPosition(newPosition);
+
+            Assert.IsTrue(result);
```

Comment:
Add verifying that position changes in database.

Reply:
Fixed